src/rust/benches/http_read.rs

```diff
@@ -0,0 +1,74 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+
+use flatgeobuf::HttpFgbReader;
+use geozero::error::Result;
+
+// 205KB
+const SMALL_URL: &str = "http://localhost:8001/countries.fgb";
+
+// 13MB
+const MEDIUM_URL: &str = "http://localhost:8001/UScounties.fgb";
+
+async fn select_all(url: &str, expected_feature_count: usize) -> Result<()> {
+    let reader = HttpFgbReader::open(url).await.unwrap();
+    let mut stream = reader.select_all().await.unwrap();
+
+    let mut count = 0;
+    while let Some(feature) = stream.next().await.transpose() {
+        let _feature = feature.unwrap();
+        count += 1
+    }
+    assert_eq!(count, expected_feature_count);
+    Ok(())
+}
+
+async fn select_bbox(url: &str, expected_feature_count: usize) {
+    let reader = HttpFgbReader::open(url).await.unwrap();
+    let mut stream = reader.select_bbox(-86.0, 10.0, -85.0, 40.0).await.unwrap();
+
+    let mut count = 0;
+    while let Some(feature) = stream.next().await.transpose() {
+        let _feature = feature.unwrap();
+        count += 1
+    }
+    assert_eq!(count, expected_feature_count);
+}
+
+fn criterion_benchmark(c: &mut Criterion) {
+    use std::time::Duration;
+    use yocalhost::ThrottledServer;
+
+    let port = 8001;
+
+    // Apply limits to simulate an "average" connection for benchmarks
+    let latency = Duration::from_millis(100);
```

Comment:
I'm happy to bikeshed these values. I tried to pick something average.

Reply:
Schould probably be an env variable.

Reply:
Good idea.

Now you can do something like this:
```
FGB_BENCH_BYTES_PER_SEC=125000000 FGB_BENCH_LATENCY_MS=50 cargo bench --bench http_read
```